docs/lib/predictions/fragments/js/sample.md

```diff
@@ -462,6 +460,502 @@ export default App;
 
 Now run `yarn start` and press the buttons to demo the app.
 
+## Sample React Native App
+
+A sample React Native application with all of the Predictions features is provided below. This shows how to use all scenarios by calling the appropriate `convert`, `identify`, and `interpret` API calls in the Amplify library.
+
+First ensure that you install React Native Expo client to run this sample app.
+
+```bash
+npm i -g expo-cli
+#Then run the following commands below to create a new React Native project called "RNAmplify" or any name of your choice.
+expo init RNAmplify
+
+cd RNAmplify
+npm start # you can also use: expo start
+
+```
+
+You will also need to install an Image Picker so as to pick images from the Gallery.
+
+```bash
+  expo install expo-image-picker
```

Comment:
We may want to see about another option here as when we add dependencies of other teams within our samples, it could lead to that dependency to be out of sync. @Ashish-Nanda Do we have the Image Picker within React Native coming soon?

Reply:
@Ashish-Nanda Can you weigh in on this.